solidity-its/src/main/java/org/sonarsource/solidity/its/SolidityRuling.java

```diff
@@ -74,17 +75,22 @@ public static void collectSolidityFiles() {
 
   public static void findDifferences() {
     filesToCompare.forEach((projectName, recordedIssues) -> {
-      String actualIssuePath = String.format("%s%s", SolidityRulingIts.ACTUAL_ISSUES,
-        recordedIssues.toString().replaceAll(SolidityRulingIts.RECORD_ISSUES, ""));
-      try {
-        if (!FileUtils.contentEquals(new File(recordedIssues.toString()), new File(actualIssuePath))) {
-          List<String> lines = Arrays.asList("Differences: " + recordedIssues.toString() + " - " + actualIssuePath);
-          Files.write(Paths.get(String.format("%s%s", SolidityRulingIts.RECORD_ISSUES, "differences")), lines, StandardCharsets.UTF_8,
-            StandardOpenOption.CREATE, StandardOpenOption.APPEND);
-        }
-      } catch (IOException e) {
-        LOG.debug(e.getMessage(), e);
-      }
+      recordedIssues.stream()
+        .forEach(issues -> {
+          String actualIssuePath = String.format("%s%s", SolidityRulingIts.ACTUAL_ISSUES,
+            issues.toString().replaceAll(SolidityRulingIts.RECORD_ISSUES, ""));
+          System.out.println("AAA: " + issues + " - " + actualIssuePath);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=35&issues=AWR-TYiFI66nRh8YLRI7&open=AWR-TYiFI66nRh8YLRI7)

[ISSUE_KEY]: AWR-TYiFI66nRh8YLRI7